codalab/apps/web/static/js/worksheet/table_bundle_interface.js

```diff
@@ -220,6 +220,12 @@ var TableBundle = React.createClass({
         });
         var focusIndex = this.state.rowFocusIndex;
         var row_items = item.interpreted[1];
+        var column_with_hyperlinks = [];
+        (Object.keys(row_items[0])).map(function(x){
```

Comment:
Professor, are you referring to use-case like``map(function(x){ `` (where parenthesis are followed by curly brackets) or just everywhere we use a {